libdnf/module/ModulePackageContainer.cpp

```diff
@@ -33,13 +33,340 @@ extern "C" {
 #include "libdnf/dnf-sack-private.hpp"
 #include <functional>
 #include <../sack/query.hpp>
+#include "libdnf/conf/ConfigParser.hpp"
+#include "libdnf/conf/OptionStringList.hpp"
 
 static std::string
 stringFormater(std::string imput)
 {
     return imput.empty() ? "*" : imput;
 }
 
+enum class ModuleState {
+    UNKNOWN,
+    ENABLED,
+    DISABLED,
+    DEFAULT
+};
+
+class ModulePackageContainer::Persistor {
+public:
+    ~Persistor() = default;
+
+    const std::string & getStream(const std::string &name);
```

Comment:
I suggest that all functions should be privat. Just to ensure that it cannot be access from outside (this is not important)

Reply:
I tried but then gcc complains:
`error: ‘bool ModulePackageContainer::Impl::ModulePersistor::insert(const string&, const char*)’ is private within this context`
even though ModulePersistor and Impl are friends.